Could not retrieve data from Experimentation service: Error: getaddrinfo ENOTFOUND default.exp-tas.com
Language server is set to "Default". Pylance is used as current language server.
Acquiring model
Acquiring model 'intellisense-members-lstm-pylance' for python
Querying IntelliCode service for available models.
vs-intellicode-python was passed a model: {}.
Couldn't reach service Error Message: getaddrinfo ENOTFOUND prod.intellicode.vsengsaas.visualstudio.com, (Error)
Could not reach the IntelliCode service. Checking the local model cache.
Recent model was found in cache.
Activating Python extension
